integration/js/checks/VideoBackgroundReplacementCheck.js

```diff
@@ -0,0 +1,27 @@
+const AppTestStep = require('../utils/AppTestStep');
+const {KiteTestError, Status} = require('kite-common');
+
+class VideoBackgroundReplacementCheck extends AppTestStep {
```

Comment:
Do we really have to create a new file for this? Can we just reuse the previous check from background blur and name this VideoBackgroundFilterCheck?

Reply:
It does have custom logic for replacement. For instance and the error message and the button to click for the test. I suppose we could take in parameters to a shared class, but then we are adding a tighter coupling between the test if we want to do something more custom in the future.

Reply:
Yeah but it can be easily say by the filter name. This does not seem very scalable when we add another video filter. There are just so many of these created (config, check, test) that are not needed.